app/models/payment.rb

```diff
@@ -190,10 +139,7 @@ def transaction_fee
     case payment_type.to_sym
     when :ideal
       return Settings.mongoose_ideal_costs
-    when :payconiq_online
-      return Settings.payconiq_transaction_costs
-    else
-      # :pin, :payconiq_display
+    when :pin
```

Comment:
Shouldn't it also be `0` on `:payconiq_online` till it is removed, as it could still be present in the database?

Reply:
good point, I'll add that